lgn-messages/src/types/v1/query/keys.rs

```diff
@@ -59,6 +69,27 @@ impl Display for ProofKey
                     KEYS_QUERIES_PREFIX, query_id, block_nr
                 )
             },
+            ProofKey::RowsChunk(query_id, key) =>
+            {
+                write!(
+                    f,
+                    "{}/{}/rows_chunk/{}/{}",
+                    KEYS_QUERIES_PREFIX,
```

Comment:
Could we define `"rows_chunk"` as a global constant, like _e.g._ `KEYS_QUERIES_PREFIX`?

Also, nitpicking, but could we put what we can directly into the format string? _E.g._ `"{KEYS_QUERIES_PREFIX}/{query_id}/{ROWS_TREE}/{block_nr}/{row_key_id}"` instead of `"{}/{}/{ROWS_TREE}/{}/{}"`